src/components/toolbox/select/index.js

```diff
@@ -49,7 +51,8 @@ class Select extends React.Component {
         <label className={`${styles.inputHolder} ${isOpen ? styles.isOpen : ''}`}>
           <Input
             readOnly
-            value={options[selected].label}
+            placeholder={placeholder}
+            value={typeof selected === 'number' ? options[selected].label : ''}
```

Comment:
💅  IMO it would be more readable and also safer (it also ensures `selected < options.length`) to say:
```suggestion
            value={options[selected] ? options[selected].label : ''}
```

Reply:
✅